lib/ansible/plugins/connection/chroot.py

```diff
@@ -103,7 +118,7 @@ def _buffered_exec_command(self, cmd, stdin=subprocess.PIPE):
         compared to exec_command() it looses some niceties like being able to
         return the process's exit code immediately.
         '''
-        executable = C.DEFAULT_EXECUTABLE.split()[0] if C.DEFAULT_EXECUTABLE else '/bin/sh'
+        executable = self.get_option('executable')
```

Comment:
I have looked at failing tests. And looks like on some environments `self.get_option('executable')` returns `None`.

Tried to look and find a root cause but it is just too much info to take in an hour :(. Quick fix would be to check if option "executable" is set, otherwise use `'/bin/sh'`

Reply:
was just missing a default